easybuild/easyconfigs/s/SSAHA2/SSAHA2-2.5.5-i686.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'SSAHA2'
+version = '2.5.5'
+versionsuffix = "_i686"
+
+homepage = 'http://www.sanger.ac.uk/resources/software/ssaha2/'
+description = """ SSAHA2 (Sequence Search and Alignment by Hashing Algorithm) is a 
+ pairwise sequence alignment program designed for the efficient mapping of sequencing 
+ reads onto genomic reference sequences. SSAHA2 reads of most sequencing platforms 
+ (ABI-Sanger, Roche 454, Illumina-Solexa) and a range of output formats (SAM, CIGAR, PSL etc.) 
+ are supported. A pile-up pipeline for analysis and genotype calling is available as 
+ a separate package. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['ftp://ftp.sanger.ac.uk/pub/resources/software/ssaha2/']
+sources = ['%(namelower)s_v%(version)s%(versionsuffix)s.tgz']
+
+sanity_check_paths = {
+    'files': ["ssaha2", "ssaha2Build", "ssahaSNP"],
+    'dirs': [""]
```

Comment:
yeah, the sync with develop can be painful, indeed...

the main reason for them is to make sure that the latest version of the unit tests are run by Jenkins (since Jenkins just checks out the branch that corresponds to the PR)

maybe we should see if we can make Jenkins merge with develop first, and then run the tests...